pkg/ccl/importccl/read_import_proc.go

```diff
@@ -376,7 +376,8 @@ func (cp *readImportDataProcessor) doRun(ctx context.Context) error {
 			defer tracing.FinishSpan(span)
 
 			writeTS := hlc.Timestamp{WallTime: cp.spec.WalltimeNanos}
-			const bufferSize, flushSize = 64 << 20, 16 << 20
+			const bufferSize = 64 << 20
+			flushSize := storageccl.MaxImportBatchSize(cp.flowCtx.Cfg.Settings)
```

Comment:
@adityamaru27 in case you missed this, I did this so that the tests that change this setting to make tiny SSTs still worked, but OTOH, it does mean we now make 32mb SSTs by default instead of 16mb. I suspect this changes our benchmark file counts somewhat -- both in that it probably makes fewer files, but *probably* also more L0 files since there's twice the chance it overlaps (particularly with workload and csv import which both use concurrent workers and thus can produce slightly ragged edges as they flush).